# XXX ../tests/helpers/test_enosys.c:25:10: fatal error: linux/seccomp.h: No such file or directory

--- a/meson.build
+++ b/meson.build
@@ -3282,11 +3282,11 @@ exes += exe
 #  exes += exe
 #endif
 
-exe = executable(
-  'test_enosys',
-  'tests/helpers/test_enosys.c',
-  include_directories : includes)
-exes += exe
+#exe = executable(
+#  'test_enosys',
+#  'tests/helpers/test_enosys.c',
+#  include_directories : includes)
+#exes += exe
 
 ############################################################
 
